project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/products-line.less

```diff
@@ -28,73 +28,42 @@
 
             text-decoration: none;
             color: @color-base;
-
-            .is-no-flex & {
-                display: block;
-                .clearfix();
-            }
         }
 
         &__position {
-            flex: 0 @list-products-line-item-postion-width;
+            width: @list-products-line-item-postion-width;
 
             font-size: 14px;
-
-            .is-no-flex & {
-                width: @list-products-line-item-postion-width;
-                float: left;
-            }
         }
 
         &__image {
-            flex: 0 @list-products-line-item-image-width;
+            width: @list-products-line-item-image-width;
             padding-right: @list-products-line-item-gap;
             text-align: center;
 
-            .is-no-flex & {
-                width: @list-products-line-item-image-width;
-                float: left;
-            }
-
             img {
                 display: block;
             }
         }
 
         &__title {
-            flex: 1;
+            width: @list-products-line-item-title-width-mobile;
             margin-bottom: 0;
             padding-left: @list-products-line-item-gap;
 
             font-size: 14px;
             font-weight: bold;
 
-            .is-no-flex & {
-                width: @list-products-line-item-title-width-mobile;
-                float: left;
-
-                @media @query-md {
-                    width: @list-products-line-item-title-width;
-                }
+            @media @query-md {
+                width: @list-products-line-item-title-width;
```

Comment:
Please check if you can substitute `flex` property for the `width` property.

Reply:
Flex: 1 sets all flexible items to be same lenght. Is not necessary bcs, we set calculated width

Reply:
It is okay for now. We will refactor this later.